### ISSUE TITLE
github.com/aws/aws-sdk-go/aws is missing from the repo dependencies

### ISSUE THREAD

**Issue by kargakis** [kubernetes/test-infra#4867]
```
$ grep -r "github.com/aws/aws-sdk-go/aws"
jenkins/aws-janitor/main.go:	"github.com/aws/aws-sdk-go/aws"
jenkins/aws-janitor/main.go:	"github.com/aws/aws-sdk-go/aws/awserr"
jenkins/aws-janitor/main.go:	"github.com/aws/aws-sdk-go/aws/session"
```

I can't find anything in Godeps.json or the vendor directory.

cc @zmerlynn

**Comment by krzyzacy** [comment]
I remember we did not include it because of the size of the aws sdk? (since we clone test-infra before each of our ci job)

**Comment by BenTheElder** [comment]
> I remember we did not include it because of the size of the aws sdk? (since we clone test-infra before each of our ci job)

Yeah, our `dep` files are now explicitly ignoring it. We could fix this by fixing #4794 and #5436 and #3369

**Comment by BenTheElder** [comment]
Alternatively we could checkout test-infra without vendor, but I really think instead of implementing this we should remove the need to checkout test-infra. 
We can put all config in config.yaml when we're done with jenkins and then we just need to put the scenarios and bootstrap into the image, and decide how we want to handle the infra-commit metadata for gubernator/test-grid.
